department-of-veterans-affairs/abd-vro
# MCP 2036 Respiratory prototype

## What was the problem?

Need to handle conditions for more respiratory disabilities. 
Associated tickets or Slack threads:

- [MCP-2036](https://amida.atlassian.net/browse/MCP-2036)

## How does this fix it?[^1]

Adding a prototype for a respiratory processor to handle emphysema, COPD and bronchitis.

## How to test this PR
- Step 1  
`
./gradlew -PenablePrototype build check docker
`

`
export COMPOSE_PROFILES=prototype
`
Run app and try fullHealthResponse endpoint.

"veteranIcn": "40000153",
  "diagnosticCode": "respiratory"
  
- Step 2[^secrel]


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
